deepchem/models/tests/test_layers.py

```diff
@@ -616,3 +616,65 @@ def test_scale_norm():
   output_ar = np.array([[5.9157897e-05, 5.8566318e-03, 5.9157896e-01],
                         [1.7754727e-06, 5.9145141e-01, 1.3611957e-02]])
   assert np.allclose(result1, output_ar)
+
+
+@pytest.mark.torch
+def test_position_wise_feed_forward():
+  """Test invoking PositionwiseFeedForward."""
+  torch.manual_seed(0)
+  input_ar = torch.tensor([[1., 2.], [5., 6.]])
+  layer = torch_layers.PositionwiseFeedForward(
+      d_input=2,
+      d_hidden=2,
+      d_output=2,
+      activation='relu',
+      n_layers=1,
+      dropout_p=0.0)
+  result = layer(input_ar)
+  output_ar = torch.tensor([[0.4810, 0.0000], [1.9771, 0.0000]])
+  assert torch.allclose(result, output_ar, rtol=1e-4)
+
+
+@pytest.mark.torch
+def test_sub_layer_connection():
+  """Test invoking SublayerConnection."""
+  torch.manual_seed(0)
+  input_ar = torch.tensor([[1., 2.], [5., 6.]])
+  layer = torch_layers.SublayerConnection(2, 0.0)
+  result = layer(input_ar, input_ar)
+  output_ar = torch.tensor([[2.0027e-05, 3.0000e+00], [4.0000e+00, 7.0000e+00]])
+  assert torch.allclose(result, output_ar)
+
+
+@pytest.mark.torch
+def test_mat_encoder_layer():
+  """Test invoking MATEncoderLayer."""
+  torch.manual_seed(0)
+  import rdkit
+  input_ar = torch.Tensor([[1., 2.], [5., 6.]])
+  mask = torch.Tensor([[1., 1.], [1., 1.]])
+  mol = rdkit.Chem.rdmolfiles.MolFromSmiles("CC")
```

Comment:
`from rdkit import Chem`
`Chem.MolFromSmiles` 

These two functions should be enough!

Reply:
Changed!